packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -328,7 +336,26 @@ export const Facet = observer((properties: FacetProps): JSX.Element => {
 							aria-level={3}
 							{...mergedLang.dropdownButton.attributes}
 						>
-							<span {...mergedLang.dropdownButton.value}>{facet?.label}</span>
+							<div>
```

Comment:
Should these two new elements be forced to be nested together in this wrapper div? Seems they could both live as siblings to the facet label? Additionally, maybe they should all be <span> elements?

Example change:
```
<div>
	<span>Color</span>
	<span class="ss__facet__header__selected-count">(3)</span>
	<span class="ss__facet__header__clear-all">...</span>
</div>
```